doc/howto/import_machines_to_instances.md

```diff
@@ -34,6 +34,30 @@ The tool can create both containers and virtual machines:
   It is also not possible to create a virtual machine from the physical machine that you are using to do the migration, because the migration tool would be using the disk that it is copying.
   Instead, you could provide a bootable image, or a bootable partition or disk that is currently not in use.
 
+   ````{tip}
+   If you want to convert a Windows VM from a foreign hypervisor (not from QEMU/KVM with Q35/`virtio-scsi`),
+   you must install the `virtio-win` drivers to your Windows. Otherwise, your VM won't boot.
+   <details>
+   <summary>Expand to see how to integrate the required drivers to your Windows VM</summary>
+   Install the required tools on the host:
+
+   1. Install `virt-v2v` version >= 2.3.4 (this is the minimal version that supports the `--block-driver` option).
+   1. Install the `virtio-win` package, or download the [`virtio-win.iso`](https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/stable-virtio/virtio-win.iso) image and put it into the `/usr/share/virtio-win` folder.
+   1. You might also need to install [`rhsrvany`](https://github.com/rwmjones/rhsrvany#full-example).
```

Comment:
The link checker can't handle anchors in GitHub links, that's a general problem. :(
I'll add an exception for that, but in this case, there's no need for the anchor (we're referring to the tool, not the example), so just remove the anchor and the link checker should be happy. :)

Reply:
>we're referring to the tool, not the example

yeah, but this example is an example of installation procedure ;-)

But sure, let's just remove anchor. Our users are administrators/programmers and they will be able to understand what to do with this link anyways.